recipes/surface-dynamics/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "surface-dynamics" %}
+{% set version = "0.4.3" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name | replace('-', '_') }}/{{ name | replace('-', '_') }}-{{ version }}.tar.gz
+  sha256: 847fdf016f3dc7d3d9f93d867ca071e90d757c7a14873ca6f04e378b75618694
+  patches:
+    - 0001-Import-sage.all-initiall-so-that-imports-resolve-cor.patch
+
+build:
+  number: 0
+  # SageMath is not packaged for Windows
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    # surface_dynamics.interval_exchanges.iet_family uses C++
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - sagelib
+  run:
+    - python
+    - sagelib
+    # LattE Integrale is recommended, but it is not packaged yet: https://github.com/conda-forge/staged-recipes/pull/8959
```

Is it worth waiting for https://github.com/conda-forge/staged-recipes/pull/8959 to be fixed and merged?
```suggestion
    - latte-integrale
```

---

From looking at the source it only adds a few features but is not essential.